packages/creator/src/createItemTemplate.ts

```diff
@@ -549,24 +551,41 @@ export function _templatizeResources(
             if (fileJson && idTest.test(dataString)) {
               const ids: string[] = dataString.match(idTest) as string[];
               const verifiedIds: string[] = [];
+              const promises = [];
+              const idLookup = [];
               ids.forEach((id) => {
+                idLookup.push(id);
                 if (verifiedIds.indexOf(id) === -1) {
                   verifiedIds.push(id);
-                  // templatize the itemId--but only once per unique id
-                  const regEx = new RegExp(id, "gm");
-                  dataString = dataString.replace(regEx, "{{" + id + ".itemId}}");
+                  promises.push(isItem(id, srcAuthentication));
+                  promises.push(isGroup(id, srcAuthentication));
+                }
```

Comment:
Interesting improvement over checking first for item and then for group or vice versa!